cpp/ql/lib/change-notes/2022-02-02-detect-creat-in-openat.md

```diff
@@ -0,0 +1,4 @@
+---
+category: minorAnalysis
+---
+* The `cpp/world-writable-file-creation` query now only detects `open` and `openat` calls with both the `O_CREAT` and `O_TMPFILE` flag, irrespective of whether the `mode` argument is present.
```

Comment:
This is getting difficult to understand.  Is the following still accurate?
```suggestion
* The `cpp/world-writable-file-creation` query now only detects `open` and `openat` calls with the `O_CREAT` or `O_TMPFILE` flag.
```

Reply:
Close enough. I'll change it.